components/KidProfile/KidProfile.styled.js

```diff
@@ -19,8 +20,47 @@ export const StyledGoBackButton = styled.button`
   border: none;
   border-radius: 4px;
   padding: 8px 16px;
+  margin: 20px;
+  cursor: pointer;
+  &:hover {
+    background-color: #6ffffa;
+  }
+`;
+export const StyledEditButton = styled.button`
+  background-color: #f0f0f0;
+  font-size: 1rem;
+  color: #333;
+  border: none;
+  border-radius: 4px;
+  padding: 8px 16px;
+  margin: 8px;
   cursor: pointer;
   &:hover {
     background-color: #6ffffa;
   }
 `;
+
+export const StyledBackLink = styled(Link)`
+  background-color: #f0f0f0;
+  font-size: 1rem;
+  color: #333;
+  text-decoration: none;
+  border: none;
+  border-radius: 4px;
+  padding: 8px 16px;
+  margin: 8px;
+  cursor: pointer;
+  &:hover {
+    background-color: #6ffffa;
+  }
+`;
+
+export const StyledList = styled.ul`
+  list-style-type: none;
+${'' /*   display: grid; */}
+${'' /*   gap: 1rem; */}
```

Comment:
It can be confusing for the future self to keep code in comments for later use. It's generally better to temove code that is commented out.